tinygrad/runtime/ops_python.py

```diff
@@ -202,4 +211,4 @@ def copyout(self, dest:memoryview, src): dest[:] = src
 
 class PythonDevice(Compiled):
   def __init__(self, device:str):
-    super().__init__(device, PythonAllocator(), PythonRenderer(), PythonCompiler(), PythonProgram)
+    super().__init__(device, PythonAllocator(), PythonRenderer(), PythonCompiler(), PythonProgram)
```

Comment:
Why is there a change here? Please read diff carefully and only have the minimum changes

Reply:
I think it might be something related to line ending characters, although I'm not certain. I don't recall making any changes to that explicitly, it just appeared. I'll make sure both of those are gone when I'm ready to merge.